### ISSUE TITLE
Move to D.py

### ISSUE THREAD

**Issue by Space-Turtle0** [School-Simplified/Timmy-SchoolSimplified#162]
### Discord Username

Discord Username: Space

### Summary

Use D.py instead of PyCord

### Feature Type

Other (Database, Voice Channels, Data Parsing)

### Current Solution

everything about Pycord

### Current Issue

everything about Pycord

### Suggested Solution

move to d.py

### Checklist

- [X] I have searched the open issues for duplicates.

**Comment by pandabear189** [comment]
https://github.com/pandabear189/Timmy-SchoolSimplified/tree/dpy-2.0

**Comment by Puncher1** [comment]
https://github.com/School-Simplified/Timmy-SchoolSimplified/tree/162-move-to-dpy

**Comment by pandabear189** [comment]
```py
@property
def display_emoji(self) -> discord.PartialEmoji:
return discord.PartialEmoji(name='...')
``` 

- [ ] add this to each cog class , with a respective emoji to show up on the help command dropdown

**Comment by pandabear189** [comment]
help command `should be done` but we will have to extensively test because it is very complex

**Comment by pandabear189** [comment]
closed #196
